### PR TITLE
Contact us component refactoring & small styling changes

### PR DESCRIPTION
**Pull request by 0750kosse**

### CODE REVIEW

File: src/components/ContactUs/index.tsx [hunk 1]
```diff
@@ -6,77 +6,61 @@ import { faEnvelope, faPhoneAlt } from '@fortawesome/free-solid-svg-icons';
...
 const ContactUsContainer = styled.div`
-display: flex;
-flex-flow: column nowrap;
-padding: 3rem .5rem;
-align-items: center;
-justify-content: center;
-width: 75%;
-margin: 0 auto;
+display:flex;
+flex-flow:row wrap;
```
Comment (t0nin0s) [inline src/components/ContactUs/index.tsx:10]: for mobile shouldnt be column?

File: src/components/ContactUs/index.tsx [hunk 2]
```diff
@@ -6,77 +6,61 @@ import { faEnvelope, faPhoneAlt } from '@fortawesome/free-solid-svg-icons';
...
-background-color: #F5F5F5;
+background:#EBEBEB;
 font-weight:700;
+box-shadow:2px 5px 5px #888888;
+${mediaQueries('tablet')`
+  position:relative;
+  top:-2rem;
+  margin: 0 auto`}
+${mediaQueries('laptop')`
+flex-flow: row nowrap;
```
Comment (t0nin0s) [inline src/components/ContactUs/index.tsx:23]: why the 'nowrap'? what would happen if you add a new service?

File: src/components/ContactUs/index.tsx (hunk 1 above)
Comment (0750kosse) [reply src/components/ContactUs/index.tsx:10]: did not like having 4 elements as column

File: src/components/ContactUs/index.tsx (hunk 2 above)
Comment (0750kosse) [reply src/components/ContactUs/index.tsx:23]: not being a designer, I would doubt you would need any other new contact option tbh...?

**Review by t0nin0s — APPROVED** [review]

**Pull request merged**